tasks/README.md

```diff
@@ -204,9 +204,14 @@ Name | Summary | Category
 `task210_logic2text_structured_text_generation.json` | Given a natural language interpretation, generate a command using logical operations | Structured Text Generation
 `task211_logic2text_classification.json` | Given a command and corresponding interpretation, classify whether it's right interpretation or not | Classification
 `task212_logic2text_classification.json` | Given a command (in the form of logical operators), classify command in one of seven logic types | Classification
+`task224_scruples_anecdotes_ethical_judgment` | Given an anecdotes, judge whether the author is ethically correct or not. | Ethical Judgment
+`task264_paper_reviews_classification` | Given a set of reviews, classify paper into accept, reject or probably reject | Classification
+`task265_paper_reviews_classification` | Given a paper review, identify it is in the english or spanish language | Classification
+`task266_paper_reviews_classification` | Given a paper review, classify into five evaluation metric | Classification
+`task267_paper_reviews_classification` | Given a paper review, classify into five orientation metric | Classification
```

Comment:
Sort the lines based on task numbers.

Reply:
What is the difference between the two lines: 
```
`task266_paper_reviews_classification` | Given a paper review, classify into five evaluation metric | Classification
`task267_paper_reviews_classification` | Given a paper review, classify into five orientation metric | Classification
```

Reply:
1. Sorted readme uploaded
2. Calculation of both the metrics is different, both indicate different things. I added an explanation in the defination.